.github/workflows/ci.yaml

```diff
@@ -12,7 +12,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node: [14, 16, 18]
```

Comment:
The Docker image still uses Node.js 14 for installing/using Auspice. There is a [PR to use Node.js 16](https://github.com/nextstrain/docker-base/pull/122) but it hasn't been pushed through since it's based on a bigger change.

Reply:
Yes, let's keep Node.js 14 support until this is done. It's mostly dependent on https://github.com/nextstrain/docker-base/pull/98, so I'll push on that a bit harder now and update here when it's done.